website/docs/advanced/override_grammar/basic.md

```diff
@@ -178,12 +182,19 @@ $ python my_app.py 'dir=job\{a\=1\,b\=2\,c\=3\}'
 - `float`: 3.14, -10e6, inf, -inf, nan.
 - `bool`: true, false
 - `dot_path`: foo.bar
-- `interpolation`: ${foo.bar}, ${co.env:USER,me}
+- `interpolation`: ${foo.bar}, ${oc.env:USER,me}
 
 Constants (null, true, false, inf, nan) are case-insensitive.
 
 :::important
-Always single-quote interpolations in the shell.
+Always single-quote interpolations in the shell, to prevent the shell from replacing them with environment variables:
```

Comment:
```suggestion
Always single-quote interpolations in the shell, to prevent the shell from replacing them with shell variables:
```

Reply:
That was a lot of repetitions of "shell" so I made it a bit shorter in 26603c5bb